.eslintrc.js

```diff
@@ -187,6 +187,10 @@ module.exports = {
         property: 'notEqual',
         message: 'Use `assert.notStrictEqual()` rather than `assert.notEqual()`.',
       },
+      {
+        property: 'webcrypto',
```

Comment:
```suggestion
        object: 'crypto',
        property: 'webcrypto',
```

Reply:
Just checked this PR locally - looks like with `object: crypto` it will prevent cases like

```js
const { webcrypto } = crypto;
```

but it does not prevent from

```js
const { webcrypto } = require('crypto').

// or 

const { subtle } = require('node:crypto').webcrypto;
```

Therefore, I think this lint rule should be ok.